### PR TITLE
AS-3650 - ClamAV Proxy and Service Common Integration

### PR DESCRIPTION
**Pull request by demzzydev**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-3650

## Description of change
Introduce ClamAV Proxy and reenable integrations with the ClamAV service-common package.

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Comment by cpcundill** [comment]
As discussed via Slack: in service-common, the package should be clamav-rest-client and in appeals the package should be clamav-rest-server

File: packages/clamav-rest-server/__tests__/fixtures/appeal.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const appealFromAppellant = {
```
Comment (cpcundill) [inline packages/clamav-rest-server/__tests__/fixtures/appeal.js:1]: Is this file used in the unit tests?

Comment (demzzydev) [reply packages/clamav-rest-server/__tests__/fixtures/appeal.js:1]: No it isn't actually, looks like hangover from a service that I copied.

File: packages/clamav-rest-server/jest-mongodb-config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (cpcundill) [inline packages/clamav-rest-server/jest-mongodb-config.js:1]: Is this file used?   The clamav-rest-server doesn't depend on mongo right?

File: packages/forms-web-app/package.json [hunk 3]
```diff
@@ -34,6 +34,7 @@
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/business-rules": "file:../business-rules",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav": "^1.0.8",
```
Comment (cpcundill) [inline packages/forms-web-app/package.json:37]: Incorrect package name and version?  Should be pins-clamav-rest-client?

File: packages/lpa-questionnaire-web-app/package.json [hunk 4]
```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav": "^1.0.8",
```
Comment (cpcundill) [inline packages/lpa-questionnaire-web-app/package.json:31]: As above - Incorrect package name and version? Should be pins-clamav-rest-client?

File: packages/forms-web-app/package.json [hunk 5]
```diff
@@ -34,6 +34,7 @@
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/business-rules": "file:../business-rules",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav-rest-client-rest-client": "^1.0.0",
```
Comment (cpcundill) [inline packages/forms-web-app/package.json:37]: Looks too long? Should be `pins-clamav-rest-client`?

File: packages/lpa-questionnaire-web-app/package.json [hunk 6]
```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav-rest-client-rest-client": "^1.0.0",
```
Comment (cpcundill) [inline packages/lpa-questionnaire-web-app/package.json:31]: Looks too long? Should be pins-clamav-rest-client?

**Review by cpcundill — CHANGES_REQUESTED** [review]

File: packages/lpa-questionnaire-web-app/package.json (hunk 6 above)
Comment (demzzydev) [reply packages/lpa-questionnaire-web-app/package.json]: Done

File: packages/forms-web-app/package.json (hunk 5 above)
Comment (demzzydev) [reply packages/forms-web-app/package.json]: Done

File: packages/lpa-questionnaire-web-app/package.json (hunk 4 above)
Comment (demzzydev) [reply packages/lpa-questionnaire-web-app/package.json]: Done

File: packages/forms-web-app/package.json (hunk 3 above)
Comment (demzzydev) [reply packages/forms-web-app/package.json]: Done

File: packages/clamav-rest-server/jest-mongodb-config.js (hunk 2 above)
Comment (demzzydev) [reply packages/clamav-rest-server/jest-mongodb-config.js]: Deleted

**Review by cpcundill — APPROVED** [review]
LGTM

**Pull request merged**